jmxtrans/docker/README.md

```diff
@@ -0,0 +1,74 @@
+# JmxTrans Docker image
+
+<img src="http://www.jmxtrans.org/assets/img/jmxtrans-logo.gif"/>
+
+Connecting the outside world to the JVM.
+
+This is a fully functional JmxTrans application instance, based on the last releaase.
+[http://www.jmxtrans.org/).
+
+
+# Usage
+
+```
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans jmxtrans/jmxtrans 
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans jmxtrans/jmxtrans start 
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans -P jmxtrans/jmxtrans jmx
+docker run -d -v `pwd`/json-files:/var/lib/jmxtrans -p 9999:2101 jmxtrans/jmxtrans jmx
+```
+
+You have two commands available :
+* start (default value)  
+* jmx to start the jvm with jmx debug mode
```

Comment:
I'm not convinced by the naming. Having a `start` and a `jmx` commands let me think that `jmx` will not start jmxtrans (when actually it does start it). Also, I'm not sure that there is really a reason to not publish JMX. Extra runtime cost is negligible and it is not exposed anyway. It might make sense to always enable JMX and to provide a default config that makes jmxtrans monitor itself.